com.amphi.music.yaml

```diff
@@ -0,0 +1,67 @@
+id: com.amphi.music
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: music
+
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/bin/lib:$LD_LIBRARY_PATH
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.portal.Desktop
+  - --socket=pulseaudio 
+
+modules:
+
+  - name: music
+    buildsystem: simple
+    build-commands:
+      - mkdir -p external
+      - tar --use-compress-program=unzstd -xf music.tar.zst
+      - tar --use-compress-program=unzstd -xf external.tar.zst -C external
+      - install -Dm755 music /app/bin/music
+      - cp -r lib /app/bin/lib
+      - cp -r data /app/bin/data
+      - ls
+      - cp -r external/* /app/bin/lib/
+
+      - install -Dm644 com.amphi.music.desktop /app/share/applications/com.amphi.music.desktop
+      - install -Dm644 icon-128.png /app/share/icons/hicolor/128x128/apps/com.amphi.music.png
+      - install -Dm644 icon-256.png /app/share/icons/hicolor/256x256/apps/com.amphi.music.png
+    
+    sources:
+      - type: file
+        dest-filename: music.tar.zst
+        only-arches: [x86_64]
+        url: https://amphi.site/releases/music/music-linux-x64-1.0.0.tar.zst
+        sha256: 37a66314529d919025c60239e903793f44d0cd2eca88330eb12f7309b47adc83
+
+      - type: file
+        dest-filename: music.tar.zst
+        only-arches: [aarch64]
+        url: https://amphi.site/releases/music/music-linux-arm64-1.0.0.tar.zst
+        sha256: eab7ea89f55fba1e19836bc6b91c5174fbe68b307b3e0e57b095c65faf73f6cb
+
+      - type: file
+        path: com.amphi.music.metainfo.xml
+      - type: file
+        path: com.amphi.music.desktop
+      - type: file
+        path: icon-128.png
+      - type: file
+        path: icon-256.png
+
+      - type: file
+        dest-filename: external.tar.zst
+        url: https://github.com/amphi2024/music/raw/refs/heads/master/linux/libs/sandbox-libraries/external_x64.tar.zst
```

Comment:
the URL isn't versioned because you are reference the tip of a git branch. This will break randomly.

(same below)